README.md

```diff
@@ -12,100 +12,124 @@ If you want to contribute or directly browse the sources of the project,
 check the [claroline/Distribution][distribution] repository, which gathers
 the standard modules and plugins of the platform.
 
-Requirements
-------------
-
-For a development installation, you'll need at least:
-
-- PHP >= 7.2,<7.4 with the following extensions:
-    - curl
-    - fileinfo
-    - [gd][gd]
-    - intl
-    - mbstring
-    - mcrypt
-    - xml
-    - json
-    - zip
-- MySQL/MariaDB >=5.0
-- [composer][composer] (recent version)
-- [node.js][node] >= 8.9
-- [npm][npm] >= 6.4
-
-It's also highly recommended to develop on an UNIX-like OS.
-
-For mysql >= 5.7, there is an additonal step:
-
-```
-    mysql -u**** -p
-    set global sql_mode='';
-    exit;
-```
-
+=======
+**/!\ Warning if you wan't to upgrade to 13.x from a older version of Claroline [read this](README.md#Upgrade-From-12x-or-older-to-13x) /!\**
 
 Installation
 ------------
 
+See Claroline [requirements here](README.md#requirements)
+
 ### 1. From source
 
-The raw installation procedure is comprised of several steps that need to be
+The raw installation procedure is composed of several steps that need to be
 executed in order (fetching php sources, installing dev dependencies, building,
 creating the database, etc.). Except for the configuration step, the whole process
 is managed through composer scripts listed in the [composer.json](composer.json)
 file. For an installation from scratch, the commands would be:
 
-    git clone http://github.com/claroline/Claroline
+    git clone -b 13.x http://github.com/claroline/Claroline
     cd Claroline
-    php scripts/configure.php
```

Comment:
Do we no longer need this command?
I see you have added it to `pre-install-cmd`, but as per composer docs:
> pre-install-cmd: occurs before the install command is executed with a lock file present
But you have gitignored the composer.lock, so I was wondering if we still need to run this command (or not) for a first-time installation?

Reply:
I've added it to .travis to add `--default` flag. Otherwise the script enters in interactive mode.
But you are right, you need to run  `php bin/configure` manually for a fresh install (I did not know the particularity of the `pre-install-cmd`).

I will make another PR to fix the doc.

Thank you !

Reply:
see https://github.com/claroline/Claroline/pull/1509.